src/components/WorkOrder/WorkOrderHeader.js

```diff
@@ -29,12 +29,7 @@ const WorkOrderHeader = ({
   }
 
   const completionReason = () => {
-    if (workOrder.status == 'Work Complete') {
-      return 'Completed'
-    }
-    if (workOrder.status == 'No Access') {
-      return 'No Access'
-    }
+    workOrder.status === 'Work Complete' ? 'Completed' : workOrder.status
```

Comment:
We need a `return` at the start of this line please, otherwise we'll be getting `undefined` back from the function

Reply:
🤦‍♀️